packages/ui/A11yDecorator/A11yDecorator.js

```diff
@@ -1,7 +1,8 @@
 /**
- * Exports the {@link ui/A11yDecorator.A11yDecorator} Higher-order Component (HOC).
+ * Exports the {@link ui/A11yDecorator.A11yDecorator} higher-order component (HOC).
```

Comment:
The new documentation style does not include links or enumerate the exports in the description.